src/accelerate/inference.py

```diff
@@ -90,7 +91,7 @@ def pippy_forward(forward, *args, **kwargs):
         if found_batch_size is None:
             raise ValueError("Could not find batch size from args or kwargs")
         else:
-            if found_batch_size != state.num_processes:
+            if found_batch_size != num_chunks:
                 args = pad_input_tensors(args, found_batch_size, state.num_processes)
                 kwargs = pad_input_tensors(kwargs, found_batch_size, state.num_processes)
```

Comment:
Do we want to pad with `num_chunks` or `state.num_processes`?

Reply:
Indeed should be `num_chunks` all the way through, bad copy/paste caught me :)